code/unclassified/create_prime_factors.c

```diff
@@ -0,0 +1,40 @@
+#include<stdio.h>
```

Comment:
The code to generate prime factor should be placed [here](https://github.com/OpenGenus/cosmos/tree/master/code/mathematical-algorithms/prime_numbers_of_n). Kindly relocate 👍